kube-runtime/src/lib.rs

```diff
@@ -10,9 +10,11 @@
 
 #![deny(clippy::all)]
 #![deny(clippy::pedantic)]
-// Triggered by many derive macros (kube-derive, derivative)
+// Triggered by many derive macros (kube-derive, educe)
 #![allow(clippy::default_trait_access)]
 #![allow(clippy::type_repetition_in_bounds)]
+// Triggered by educe derives on enums
+#![allow(clippy::used_underscore_binding)]
```

Comment:
In response to the PR body, yeah, I think this is acceptable. Kind of doomed to happen with pedantic lints, but that's another discussion :D